unfccc.csl

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UNFCCC</title>
+    <title-short>UNFCCC</title-short>
+    <id>http://www.zotero.org/styles/unfccc</id>
+    <link rel="self" href="http://www.zotero.org/styles/unfccc"/>
+    <link href="https://www.zotero.org/styles/harvard-kings-college-london" rel="template"/>
```

Comment:
> It's primarily for use by authors of UNFCCC documents in Zotero or another CSL-based ref. manager.

Okay, thanks. That's a good fit for this repository, then.

> We don't have a public guide for references

Is there a private one that is cataloged somehow? We always try to link to a document that describes the citation format, even if it's not publicly accessible.